data-collection/utils/version.json

```diff
@@ -1,3 +1,3 @@
 {
-    "version": "3.0.7"
+    "version": "3.0.8"
```

Comment:
Per above, version bump is automated.

Reply:
@ftresgallo if you agreed with @esc1144 to revert version, this must be reverted as well. 
 
 For me integrating on automatic bump commit to the feature is ok as well. But partial revert can be problematic. So just revert this one for consisttency 
 
 
 also @esc1144  please create another PR with automated release bump. You will need someone to approve while it would be in initial PR we could have just one approver